tests/orbopt/test_gasscf.py

```diff
@@ -1,123 +1,184 @@
+from logging import fatal
 import pytest
 
-from forte2 import System, RHF, MCOptimizer, State, CI, ROHF
+from forte2 import *
 from forte2.helpers.comparisons import approx
 
-
 @pytest.mark.skip
```

Comment:
would be good to add a reason here for future reference (`@pytest.mark.skip(reason="falls below forte reference results")`)